spec/filters/grok_spec.rb

```diff
@@ -1,5 +1,13 @@
 # encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+
+# running the grok code outside a logstash package means
+# LOGSTASH_HOME will not be defined, so let's set it here
+# before requiring the grok filter
+unless LogStash::Environment.const_defined?(:LOGSTASH_HOME)
+  LogStash::Environment::LOGSTASH_HOME = File.expand_path("../../../", __FILE__)
```

Comment:
[const_defined?](http://ruby-doc.org/core-1.9.3/Module.html#method-i-const_defined-3F) takes a symbol, otherwise it would raise exception when evaluating the function's arguments